uniffi/src/bindings/python/templates/RustBufferHelper.py

```diff
@@ -44,7 +44,14 @@ def getInt(self):
         return self._unpack_from(4, ">i")
 
     def putInt(self, v):
-        self._pack_into(8, ">i", v)
+        self._pack_into(4, ">i", v)
+
+    def getString(self):
+        len = self.getInt()
+        return self._unpack_from(len, ">{}s".format(len(s))).decode('utf-8')
```

Comment:
I don't understand what `len(s)` does here, should it just be `len`? (Which stylistically would be better to name like `numBytes` or something to avoid overwriting the builtin `len` function).